src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -450,4 +450,55 @@ describe('atomicallyReadWriteResources', () => {
             await runUpdateTest(true);
         });
     });
+
+    describe('Update as Create Cases', () => {
+        const runTest = async (supportUpdateCreate: boolean, operation: TypeOperation, isLockSuccessful: boolean) => {
+            // READ items (Failure)
+            AWSMock.mock('DynamoDB', 'query', (params: QueryInput, callback: Function) => {
+                callback(null, { Items: [] });
+            });
+
+            const dynamoDb = new AWS.DynamoDB();
+            const bundleService = new DynamoDbBundleService(dynamoDb, supportUpdateCreate);
+
+            const updateRequest: BatchReadWriteRequest = {

```

Comment:
This could be a non-update request right?

```suggestion
            const batchRequest: BatchReadWriteRequest = {

```